opm/simulators/wells/MultisegmentWell_impl.hpp

```diff
@@ -577,21 +577,28 @@ namespace Opm
             if (eq_idx < num_components_) { // phase or component mass equations
                 const double flux_residual = maximum_residual[eq_idx];
                 // TODO: the report can not handle the segment number yet.
+
+                double relax_factor = 1.0;
+                if (relax_tolerance)
+                    relax_factor = 1.e4;
```

Comment:
With this, basically we do not care much about the flux residual. Similar to that we ignore the CNV checking after 8 iterations.

Reply:
Does this mean accept anything? If this "tunings" is a good idea, I think they should be paramters forexample --mswells-relaxed-convergence-factor=1e4 --mswells-volume-factor=10, and that the default values are 1.0 for bouth.

Reply:
Agree @hnil. We just need to test this on more models first to see if it is a good idea. We may need to change the tolerances a bit after testing. The tolerances here are just tuned to remove all convergence issues for MSW on the 9_4E_...MSW model. If we can accept 10+ convergence issues we can tighten these numbers.

Reply:
> Does this mean accept anything? 

It will be tolerance within 86400. m^3/day for the flux tolerance. It is not anything, but almost. The flux residual with this value will not be right.

Reply:
> If this "tunings" is a good idea, I think they should be paramters forexample --mswells-relaxed-convergence-factor=1e4 --mswells-volume-factor=10, and that the default values are 1.0 for both.

This is a good suggestion. Then people can create tests with running time arguments without changing default behavior.

Reply:
The tolerance relaxation is only in the inner iterations. The strict tolerance is still used when getWellConvergence is called during the main newton loop. The change of tolerance here should therefore not effect the end results only the road towards it. I don't think we can expect the MSW to be able to converge fully early in the newton iterations before the near well solution starts to settle a bit.

Reply:
I think this is done in a good way, and agree that the main convergence check will never see any relaxation.